mlflow/pyfunc/model.py

```diff
@@ -292,6 +295,9 @@ def _save_model_with_class_artifacts_params(
     if mlflow_model is None:
         mlflow_model = Model()
 
+    if python_model is None:
+        _logger.warning("The provided Python model is None. The model will be saved as None.")
```

Comment:
This is true for all models logged as code right? Since when loading the model we would only have a model_code_path?

So do we need this warning?

Reply:
Sorry this was for debugging 😭 . Forgot to remove it.